priam/src/main/java/com/netflix/priam/config/CassandraConfiguration.java

```diff
@@ -29,6 +29,9 @@
     private String cassHome;
 
     @JsonProperty
+    private String yamlLocation;
```

Comment:
`@JsonIgnore` for this property. So, we can use the default if yamlLocation is not provided (in production system)